packages/nuxeo-retention-package/src/main/resources/install/templates/retention/nxserver/config/retention-compliance-config.xml.nxftl

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+
+<component name="org.nuxeo.retention.compliance.types">
+<#if "${nuxeo.retention.compliance.enabled}" == "true">
+    <require>org.nuxeo.retention.types</require>
+    <require>org.nuxeo.ecm.core.schema.TypeService</require>
```

Comment:
This `<require>` is useless, it's implicit in the fact that we use extension points from this component.

Reply:
What about loading order as we discussed [above](https://github.com/nuxeo/nuxeo/pull/3857#discussion_r397848731)? Is there any chance that this specific contribution is loaded before `org.nuxeo.retention.types`?

Reply:
point it to keep `<require>org.nuxeo.retention.types</require>` but drop the `<require>org.nuxeo.ecm.core.schema.TypeService</require>`  since you're already making contributions to it